docker-compose-production.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.4'
+services:
+  web:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+      target: production
+    depends_on:
+      - db
+    ports:
+      - "${WEB_PORT:-80}:8000"
+    environment:
+      NODE_ENV: production
+      MONGO_URL: mongodb://db:27017/mern-starter
+  db:
+    image: mongo:latest
+    ports:
+      - "27017:27017"
```

Comment:
Should we map the mongodb port here? It's not necessary and could potentially conflict with users who have mongodb running locally.

(edit: I had thrown it in here when the test suite required a real mongodb instance to be running… it no longer does)